irods/rule.py

```diff
@@ -21,6 +29,17 @@ def __init__(self, session, rule_file=None, body='', params=None, output=''):
         if output != '':
             self.output = output
 
+    #define RULE_EXEC_DEL_INP_PI "str ruleExecId[NAME_LEN];"
```

Comment:
unneeded?

Reply:
Not needed, but (often in precedent) provided for reference.